db/fixtures/chargeback_rates.yml

```diff
@@ -17,6 +17,18 @@
           :finish: Infinity
           :fixed_rate: 0.0
           :variable_rate: 0.02
+    - :description: Used CPU Cores
+      :group: cpu
+      :source: used
+      :metric: cpu_usage_rate_average
```

Comment:
So we can investigate this in: #7081 since I believe it works correctly.
but over all, I didnt have to make special logic for this. it is similar enough to ```cpu_usagemhz_rate_average``` to work without changes. This is grouped in used_cpu and used_cpu_cost just like the mhz assignment. 
We cant measure container usage by mhz so think this change should be fine.

check out the logic here: https://github.com/ManageIQ/manageiq/blob/master/app/models/chargeback.rb#L164